### PR TITLE
Handle null values of AddMapConfigMessageTask's nullable fields

### PR DESCRIPTION
**Pull request by srknzl**
This PR adds necessary null checks to the fields of `AddMapConfigMessageTask` that are mistakenly marked as nullable in the client protocol which are actually not nullable.

backport of #21417

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java [hunk 1]
```diff
@@ -76,17 +76,17 @@ protected AbstractMessageTask(ClientMessage clientMessage, Node node, Connection
         this.clientMessage = clientMessage;
         this.logger = node.getLogger(getClass());
         this.node = node;
-        this.nodeEngine = node.nodeEngine;
```
Comment (zpqrtbnk) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java:79]: Are these changes related to the rest of the PR?

**Review by zpqrtbnk — COMMENTED** [review]
Have 1 question.

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java (hunk 1 above)
Comment (srknzl) [reply hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java:79]: Not much related, but I guess the person who did the fix fixed these along the way. I guess they are to use getters instead of using fields directly.

**Review by zpqrtbnk — APPROVED** [review]
Approved
